ios/Runner/Info.plist

```diff
@@ -28,6 +28,17 @@
 	<string>To get location and direction</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>To get location and direction</string>
```

Comment:
I think when uploading to app store, these reasons will cause a rejection, cause apple is so picky regarding the permissions reasons. try providing a more descriptive reason cause this string is the one that will be shown to the user.

Reply:
done